README.md

```diff
@@ -93,26 +106,21 @@ Open up [Swagger](http://localhost/docs) to see interactive API.
 
 To debug frontend, you simply open the **Run and Debug** panel and select **Debug Frontend**.
 
-To debug backend, start by configuring Poetry to create virtual environment within the project folder.
+To debug backend, you have to set your Python Interpreter path to your virtual environment
 
 Go to backend directory:
 ```
 cd backend
 ```
-
-Check installed poetry environment by running
-```
-poetry env info
-```
-This will list path to the virtual environment created by Poetry:
+Find the path to the python executable in the virtual environment by running `poetry env info`
 ```
 ➜  backend git:(vscode-debug) ✗ poetry env info
 
 Virtualenv
 Python:         3.12.5
 Implementation: CPython
-Path:           /home/ilhanbae/.cache/pypoetry/virtualenvs/app-6zZbMuyK-py3.12
-Executable:     /home/ilhanbae/.cache/pypoetry/virtualenvs/app-6zZbMuyK-py3.12/bin/python
+Path:           /home/mitchell/ComputerScience_WSL/Incredihire/Survey/backend/.venv
```

Comment:
This probably shouldn't be in here, right?

Reply:
The intention of this is to show how to find the path to the virtual environment

Reply:
fixed